lib/msal-node/src/client/ManagedIdentitySources/AzureArc.ts

```diff
@@ -194,9 +193,9 @@ export class AzureArc extends BaseManagedIdentitySource {
 
         request.headers[METADATA_HEADER_NAME] = "true";
 
-        request.queryParameters[API_VERSION_QUERY_PARAMETER_NAME] =
-            ARC_API_VERSION;
-        request.queryParameters[RESOURCE_BODY_OR_QUERY_PARAMETER_NAME] =
+        request.queryParameters[ManagedIdentityQueryParameters.API_VERSION] =
+            MSI_V1_MIN_VERSION;
```

Comment:
Service Fabric will be the only resource that supports Token Revocation for now. For the private preview let's only focus on Service Fabric and not change the api-version for other Azure resources